politeiawww/cmd/piwww/piwww.go

```diff
@@ -45,62 +45,67 @@ type piwww struct {
 	Config shared.Config
 
 	// Proposal commands
-	ProposalNew       ProposalNewCmd       `command:"proposalnew"`
-	ProposalEdit      ProposalEditCmd      `command:"proposaledit"`
-	ProposalSetStatus ProposalSetStatusCmd `command:"proposalsetstatus"`
-	Proposals         ProposalsCmd         `command:"proposals"`
-	ProposalInventory ProposalInventoryCmd `command:"proposalinventory"`
+	ProposalNew       proposalNewCmd       `command:"proposalnew"`
+	ProposalEdit      proposalEditCmd      `command:"proposaledit"`
+	ProposalSetStatus proposalSetStatusCmd `command:"proposalsetstatus"`
+	Proposals         proposalsCmd         `command:"proposals"`
+	ProposalInventory proposalInventoryCmd `command:"proposalinventory"`
+	ProposalsVetted   proposalsVettedCmd   `command:"proposalsvetted" description:"(public) get a page of vetted proposals"`
```

Comment:
why is there a seperate command for vetted proposals?

Reply:
It was already there so I thought of leaving it, but indeed it doesn't make sense anymore to have that cmd. Removing it